### DISCUSSION TITLE
Whats difference between kirin 659 a and 659 b

### DISCUSSION THREAD

**Discussion by tahirjamil** [kitsuned/PotatoNV#140] [Q&A] [answered]
I have honor 9n (9i) , it uses kirin 659 , but the potato nv has 2 options 1. Kirin 659 a, 2. Kirin 659 b. I dont know what should it mean and what should i use for mine

**Comment by mashed-potatoes** [comment, marked as answer]
These are different revisions of the SoC.
Just try both options. The wrong bootloader won't harm your device.
Please let me know the result. I will put the model in table.

**Reply by tahirjamil** [reply]
My one is Honor 9N, it worked with kirin 659 A
